src/components/localization/language-dropdown.js

```diff
@@ -171,10 +172,11 @@ const query = graphql`
     }
 `
 
-const Dropdown = ({ default_option, onChange, option_list, is_high_nav }) => {
+const Dropdown = ({ default_option, onChange, option_list, is_high_nav, security }) => {
     const [is_open, setOpen] = React.useState(false)
     const dropdown_ref = React.useRef(null)
-
+    const [is_mobile] = useBrowserResize()
+    const iconSize = security && is_mobile ? '20px' : '24px'
```

Comment:
```suggestion
    const icon_size = security && is_mobile ? '20px' : '24px'
```
Variables should follow the snake_case convention.